package exception;

public class classA {
	
	public void func() throws Exception
	{
		/*try{
			
		}
		catch(Exception ex)
		{}*/
		throw new Exception("Sorry");
		
		
	}
	

	

}
